medium/__init__.py

```diff
@@ -7,84 +7,139 @@
 
 import requests
 
-BASE_PATH = "https://api.medium.com"
 
+def _request(method, path, access_token, json=None, form_data=None,
```

Comment:
The reduction in complexity is especially helpful for someone reading this code for the first time, even if it makes things more verbose, so I think we should fight for it if we can.

It does look like a slippery slope, but I don't think it really is. I'm certainly not advocating that every method replace its body with a call to a private function. Many can't do that anyway if they change the state of the object. And if the body is never duplicated, by splitting it out we're only adding an unhelpful level of indirection and making things more complex. But for cases where we're reusing logic and we aren't using inheritance, I think the reduction in complexity is worth the costs of verboseness and indirection.

In this case, I think it's especially right, since the idea of "how do I make a request to the Medium API" and "how do I request specific resources" are two separate responsibilities.